src/main/java/capstone/mfslbackend/model/FantasyTeam.java

```diff
@@ -33,15 +32,16 @@ public class FantasyTeam {
     private Long id;
     private String teamName;
     private String playerIdsInOrder;
+    private int orderNumber;
+    private int opponentNumber;
 
-    @OneToMany(mappedBy = "fantasyTeam", orphanRemoval = true)
-    private Set<FantasyWeek> fantasyWeeks = new LinkedHashSet<>();
+    @OneToMany(mappedBy = "fantasyTeamA", orphanRemoval = true)
+    private Set<FantasyWeek> fantasyWeeksA = new LinkedHashSet<>();
```

Comment:
If possible, we should have only one list of fantasyWeeks for a fantasyTeam. Yes I understand it's a bit harder because there are two FantasyTeams in a FantasyWeek. Basically instead of having two OneToMany associations that are mapped by fantasyTeamA and fantasyTeamB, this list should not know about the fantasyTeamA and fantasyTeamB in the fantasyWeek